composer.json

```diff
@@ -22,14 +22,14 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0",
+        "php": "^7.4 || ^8.0",
         "doctrine/persistence": "^1.3 || ^2.0",
-        "sonata-project/classification-bundle": "^3.12",
+        "sonata-project/classification-bundle": "^4.0@dev",
         "sonata-project/doctrine-extensions": "^1.10.1",
         "sonata-project/form-extensions": "^1.4",
         "sonata-project/formatter-bundle": "^4.0",
         "sonata-project/intl-bundle": "^2.4",
-        "sonata-project/media-bundle": "^3.20",
+        "sonata-project/media-bundle": "^4.0@dev",
```

Comment:
We have to first release the major version of the media bundle and the classification bundle then

Reply:
Agree. Anyway we can add `^4.0@dev` or `4.x-dev` in unstable branch here to allow work on Sonata 4 in all sonata dependencies at once. We have to remove it before stable release.

Reply:
Sure